x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_unified_search.ts

```diff
@@ -11,17 +11,32 @@ import { buildEsQuery, Filter, Query, TimeRange } from '@kbn/es-query';
 import type { SavedQuery } from '@kbn/data-plugin/public';
 import { debounce } from 'lodash';
 import deepEqual from 'fast-deep-equal';
+import moment from 'moment';
 import type { InfraClientStartDeps } from '../../../../types';
 import { useMetricsDataViewContext } from './use_data_view';
 import { useSyncKibanaTimeFilterTime } from '../../../../hooks/use_kibana_timefilter_time';
-import { useHostsUrlState, INITIAL_DATE_RANGE } from './use_unified_search_url_state';
+import { useHostsUrlState, INITIAL_DATE_RANGE, HostsState } from './use_unified_search_url_state';
+
+const buildQuerySubmittedPayload = (hostState: HostsState) => {
+  const { panelFilters, filters, dateRangeTimestamp, query: queryObj } = hostState;
+
+  return {
+    control_filters: panelFilters.map((filter) => JSON.stringify(filter)),
+    filters: filters.map((filter) => JSON.stringify(filter)),
+    interval: moment
```

Comment:
⚡ !

Reply:
About the date, we were at first tracking the interval as `from:now-3d,to:now` which didn't seem really readable, so we switched to this approach. It is helpful for us to see what time range users prefer to get an overview of, @roshan-elastic do you have any preference about this?

About the JSON stringified content, these filters reduce to es-query and it's difficult to extract a human-readable version of them.

Reply:
I guess the main goal here is to understand how wide, in average, will be the ranges in the searches. The idea is to understand whether we can limit the interval in the search bar, to, let's say, up to 7 days. Because if we let users to look back 2 months worth of data, it impacts the query performance.

I'm not sure how relevant it is to know if they are searching for `25 days ago` or any `25 days` interval in the past , @roshan-elastic what are your thoughts on this? 

The way the string will be sorted seems bad, but I'm not sure how that impacts in the analysis.

> About the date, we were at first tracking the interval as from:now-3d,to:now 

When using absolute dates, it could be, for instance: `from:2023-01-24T10:06:00Z,to:now`, IMO it would be harder to read this in FullStory

Reply:
Hey @crespocarlos - I think you're right.

I know this wasn't defined in the acceptance criteria so I'll get what I'm given but to try and help, here is some info:

**Primary goal with dates**
The main goal is to understand the date range people use to help us understand if we can cut down the available range to a shorter period (e.g. do we need more than 2 days available)?

- I'd expect to a field in the elastic telemetry cluster (or fullstory) which shows me the time range as a single field (sorting asc would have the smallest time range, e.g. milliseconds - sorting desc would show the largest, e.g. days/years). This could be just be the from>to unix timestamps subtracted (to give me the ms between the two?)

- It would be awesome if we could classify these into buckets:

For example: 

   1. Less than 1 minute
   2. 1-5 minutes
   3. 5-10 minutes
   4. 11-30 minutes
   5. 30-60 minutes
   6. 1-2 hours
   7.  etc

Then, I could easily rank the above to get a very simple view on the length of time range people have (I could correlate that with the actual time difference if I wanted to go more granular). It could also build easy charts with the above kind of data (although having the granularity of the actual ms between from/to would be great too for more detailed analysis).

**Secondary goal with dates**
How long ago are they searching?

If I could correlate the above with whether it was a more recent vs further away - that'd be useful but not sure how useful right now.

Ideally, I'd like to understand whether people are looking at old data or new data (and maybe that would inform the UI/optimisations we can make).

Not sure if this helps?

Reply:
Thanks for this answer @roshan-elastic, I like the idea to send the difference in milliseconds for the timestamps, it can definitely give the most granular control you need! In terms of the ranges we should maybe define them more specifically, I'll pass the difference of milliseconds for this first iteration and we'll update this part if necessary for a deeper analysis.

Reply:
Sure thing @tonyghiani!

This might not be the best place to define them but sticking them here...happy to move if there's a better place:

1. Less than 1 minute
2. 1-5 minutes
3. 5-10 minutes
4. 11-30 minutes
5. 31-60 minutes
6. 1-2 hours
7. 3-8 hours
8. 9-12 hours
9. 13-24 hours
10. 1-2 days
11. 3-7 days
13. 8-14 days 
14. 15-30 days 
15. 31-60 days
15. 61-90 days
16. 91-180 days
17. 180-365 days
18. More than 1 year

Reply:
Perfect, I'll update the PR with this piece of logic!